modules/lti/src/main/resources/tools/courses/js/lti.recordings.js

```diff
@@ -208,9 +208,14 @@ $(document).on("click", ".downloader", function () {
         month = timestamp.getMonth() < 9 ? '0' + (timestamp.getMonth() + 1) : timestamp.getMonth() + 1,
         day = (timestamp.getDate() < 10 ? '0' : '') + timestamp.getDate(),
         dateStamp = "" + timestamp.getFullYear() + month + day;
-
+   
+    if(episodePresenter === "undefined"){
```

Comment:
Please use spaces around brackets (before open and after close)

    if (episodePresenter === "undefined") {